src/widgets/restFullClient/ui/MethodManager.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+} from '@mui/material';
+import { ICRUD, IMethodProps } from '../model/methodManagerModel';
+import { FC } from 'react';
+
+const MethodManager: FC<IMethodProps> = ({ method, setMethod }) => {
+  const handleChange = (e: SelectChangeEvent<string>): void => {
+    const { value } = e.target;
+    setMethod(value as ICRUD);
```

Comment:
Добавляй проверки на всякйи случай
```suggestion
  const handleChange = (e: SelectChangeEvent<string>): void => {
    const { value } = e?.target || {};
    
    if (!value) {
       return
    }
    
    setMethod(value as ICRUD);
```

Reply:
добавил проверку